.gitlab/kernel_version_testing/system_probe.yml

```diff
@@ -131,24 +152,34 @@ kernel_matrix_testing_setup_env:
       - $LibvirtSSHKeyARM
       - $LibvirtSSHKeyARM.pub
       - $STACK_DIR
+      - $CI_PROJECT_DIR/stack.outputs
+      - $AWS_EC2_SSH_KEY_FILE
 
 kernel_matrix_testing_upload_deps:
   extends:
     - .kitchen_ec2_location_us_east_1
   stage: kernel_matrix_testing
-  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/system-probe_x64:$DATADOG_AGENT_SYSPROBE_BUILDIMAGES
-  needs: ["kernel_matrix_testing_setup_env", "package_dependencies_x64", "package_dependencies_arm64"]
-  tags: ["runner:main"]
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/test-infra-definitions/runner:7c39524b
```

Comment:
Should the image hash be a variable?

Reply:
The E2E squad is using a different version currently. I am can ask them if they would rather move to a global variable.